source/includes/_datasets.md

```diff
@@ -105,12 +104,52 @@ This endpoint lists all the datasets that can be edited by this user.
 Parameter | Default | Description
 --------- | ------- | -----------
 `where` <br> *string* | None | Filter expression used to restrict returned datasets ([ODSQL documentation](https://docs.opendatasoft.com/api/explore/v2.html#where-clause))
-`start` <br> *string* | 0 | Index of the first item to return
+`page` <br> *string* | 1 | Number of the page you want to retrieve. Pages numbers vary when changing the number of rows to retrieve.
 `rows` <br> *string* | 10 | Number of items to return. Max value: 100
 `sort` <br> *string* | None | Field on which to sort the results list
 `include_app_metas` <br> *string* | false | Explicitely request application metadata for each datasets
 `timezone` <br> *string* | UTC | Timezone applied on datetime fields in query and response
 
 ### Returns
 
-Returns a list of dataset objects.
\ No newline at end of file
+Returns a list of dataset objects.
+
+
+## Retrieve information about one dataset
+
+This endpoint is for retrieving the dataset object with provided dataset_uid.
+
+> Definition
+
+```HTTP
+GET https://{DOMAIN_ID}.opendatasoft.com/api/management/v2/datasets/{DATASET_UID}
+```
+
+> Example request
+
+```shell
+curl 'https://yourdomain.opendatasoft.com/api/management/v2/datasets/da_7jgvnj'
+```
+
+> Example response
+
+```json
+{
+    "dataset_id": "my-dataset",
+    "dataset_uid": "da_7jgvnj",
+    "metas": {
+        "default": {
+            "modified": "2017-08-27T20:17:30+00:00",
+            "language": "en",
+            "title": "My Dataset",
+        }
+    },
```

Comment:
Isn't metas an expandable property ? If so the URL should reflect that with an "expand" parameter.

Reply:
true